tests/backup/backup_helper.go

```diff
@@ -867,14 +867,19 @@ func CleanupCloudSettingsAndClusters(backupLocationMap map[string]string, credNa
 			_, err = task.DoRetryWithTimeout(backupLocationDeleteStatusCheck, cloudAccountDeleteTimeout, cloudAccountDeleteRetryTime)
 			Inst().Dash.VerifySafely(err, nil, fmt.Sprintf("Verifying backup location deletion status %s", bkpLocationName))
 		}
+		err := DeleteCloudCredential(credName, orgID, cloudCredUID)
+		Inst().Dash.VerifySafely(err, nil, fmt.Sprintf("Verifying deletion of cloud cred [%s]", credName))
 		cloudCredDeleteStatus := func() (interface{}, bool, error) {
-			err := DeleteCloudCredential(credName, orgID, cloudCredUID)
-			if err != nil {
-				return "", true, fmt.Errorf("deleting cloud cred %s", credName)
+			status, err2 := IsCloudCredPresent(credName, ctx, orgID)
```

Comment:
Declare `var err error` in the function and keep using err everywhere

Reply:
Done